flutter_local_notifications/ios/Classes/FlutterLocalNotificationsPlugin.m

```diff
@@ -912,14 +914,40 @@ - (void)cancelAll:(FlutterResult _Nonnull)result {
             [NSMutableArray arrayWithCapacity:attachments.count];
         for (NSDictionary *attachment in attachments) {
           NSError *error;
+
+          NSMutableDictionary *options = [[NSMutableDictionary alloc] init];
+          if ([self containsKey:ATTACHMENT_HIDE_THUMBNAIL
+                  forDictionary:attachment]) {
+            NSNumber *hideThumbnail = attachment[ATTACHMENT_HIDE_THUMBNAIL];
+            [options
+                setObject:hideThumbnail
+                   forKey:UNNotificationAttachmentOptionsThumbnailHiddenKey];
+          }
+          if ([self containsKey:ATTACHMENT_THUMBNAIL_CLIPPING_RECT
+                  forDictionary:attachment]) {
+            NSDictionary *thumbnailClippingRect =
+                attachment[ATTACHMENT_THUMBNAIL_CLIPPING_RECT];
+            CGRect rect =
+                CGRectMake([thumbnailClippingRect[@"x"] doubleValue],
+                           [thumbnailClippingRect[@"y"] doubleValue],
+                           [thumbnailClippingRect[@"width"] doubleValue],
+                           [thumbnailClippingRect[@"height"] doubleValue]);
+            NSDictionary *rectDict =
```

Comment:
for consistency and to avoid any potential issues down the line given what we saw on macOS, can this go back to using dictionaryRepresentation like it did before?

Reply:
I'm not sure if I understand... This part was always like that. The `dictionaryRepresentation` method on `CGRect` is there in Swift, but in ObjC this is it.

Reply:
Apologies misremembered and turns out it's not even available in Objective-C